addon/components/frost-numeric-select.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * Component definition for the numeric selection field component
+ */
+import layout from '../templates/components/frost-numeric-select'
+import Component from './frost-component'
+import Ember from 'ember'
+import {PropTypes} from 'ember-prop-types'
+import computed, {readOnly} from 'ember-computed-decorators'
+const {isPresent} = Ember
+
+export default Component.extend({
+  // == Dependencies ==========================================================
+
+  // == Keyword Properties ====================================================
+  layout,
+
+  // == PropTypes =============================================================
+
+  propTypes: {    // options
+    align: PropTypes.string,
+    isHookEmbedded: PropTypes.bool,
+    receivedHook: PropTypes.string,
+    tabindex: PropTypes.number,
+    type: PropTypes.string,
+    isArrowVisible: PropTypes.bool,
+    isAllowInputNegative: PropTypes.bool,
+    isAllowUseDownButtonToNegative: PropTypes.bool,
+    showError: PropTypes.bool,
+    errorMessage: PropTypes.string,
+    class: PropTypes.string,
+
+    // Setting these as part of establishing an inital value
+    autocapitalize: PropTypes.string,
+    autofocus: PropTypes.bool,
+    autocorrect: PropTypes.string,
+    form: PropTypes.string,
+    maxlength: PropTypes.number,
+    placeholder: PropTypes.string,
+    required: PropTypes.bool,
+    selectionDirection: PropTypes.string,
+    spellcheck: PropTypes.bool,
+    value: PropTypes.string,
```

Comment:
Ember (and other frameworks) promote a concept known in Ember as Data Down, Actions Up https://dockyard.com/blog/2016/11/18/checkbox-list-ember

Basically the idea is that the state required for a component to function should be held outside of the component and component events should fire to that layer to mutate the state.

This means that `value` is your Data Down and you need a callback like `onInput` to act as the Action Up.

A consumer would have a property that would store the actual state, something like 

https://github.com/ciena-frost/ember-frost-core/blob/master/tests/dummy/app/pods/toggle/template.hbs#L12
and
https://github.com/ciena-frost/ember-frost-core/blob/master/tests/dummy/app/pods/toggle/controller.js#L23